sysconfig/export_test.go

```diff
@@ -31,4 +31,12 @@ func CloudDatasourcesInUse(configFile string) (*CloudDatasourcesInUseResult, err
 	return (*CloudDatasourcesInUseResult)(res), err
 }
 
+func CloudDatasourcesInUseForDir(dir string) (*cloudDatasourcesInUseResult, error) {
```

Comment:
should the return use CloudDatasourcesInUseResult ? otoh it doesn't seem to matter (golang is ok accessing exported fields of unexported types as long as the latter doesn't need to be explicitly referenced)

Reply:
yes sorry this was a mistake to return the unexported type here